source/logs.txt

```diff
@@ -0,0 +1,49 @@
+.. _intellij-logs:
+
+======================
+Retrieve IntelliJ Logs
+======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+This page describes how to retrieve log files from IntelliJ IDEA.
+
+About This Task 
+---------------
+
+Log files help the MongoDB development team diagnose bugs in the 
+{+intellij-short+}. When you retrieve log files, IntelliJ generates a zip with 
+the following contents:
+
+- General information about the IDE installation such as version, host OS, host 
+  CPU, available memory, other plugins that might affect current functionality.
+- Thread dumps of the current state of IntelliJ. 
+- Internal state of the UI.
+- OpenTelemetry metrics from IntelliJ.
+- Logs of all the plugins, including the {+intellij-full+}.
+
+Steps
+-----
+
+.. procedure:: 
+   :style: normal
+
+   .. step:: In the IntelliJ top menu bar, click Help.
+
+   .. step:: In the Help menu, click Collect Logs and Diagnostic Data. 
+     
+      A pop-up warning appears, mentioning that the collected information might 
+      contain sensitive data. 
+      
+   .. step:: Click Show in Finder.
```

Comment:
[question]

Should we include any Windows-specific instructions here? Is this supported on Windows?

Reply:
Let me double check with Gaurab on that! I didn't consider the different Windows elements 😅

Reply:
@GaurabAryal For the warning that pops up when downloading the logs, does the UI have different text for Window/Linux users as opposed to macOS users? 

Also would it be worth mentioning if there's a default file path that these logs are sent to for each OS?

Reply:
Hmm that is a good question, flagging for @kmruiz.

Reply:
I can actually try it for both operating systems. Let me take a look and post the messages for both OS.

Reply:
I confirm the messaging is the same for Linux and Windows. Tested in Windows 11 and Debian + Cinnamon.